cmd/opentelemetry-prometheus-sidecar/e2e_test.go

```diff
@@ -267,12 +267,10 @@ func TestE2E(t *testing.T) {
 			rvals[attr.Key] = attr.Value.Value.(*common.AnyValue_StringValue).StringValue
 		}
 
+		// At this moment, the labels in static_configs are NOT
+		// passed to the Resource.
 		if diff, equal := messagediff.PrettyDiff(rvals, map[string]string{
 			"service.name": "Service",
-			"instance":     "127.0.0.1:19002",
```

Comment:
I think these tests should move -- IIUC these become labels. There were no labels being tested here because there are no labels in the e2e target (iirc).